src/app/meshes/views/MeshView.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <KTabs
+    :tabs="items"
+    :model-value="(items.find(item => (router.currentRoute?.value.name ?? '').toString().startsWith(item.hash)) ?? items[0]).hash"
+    @click="change"
+  >
+    <template #mesh-detail-view-anchor>
```

Comment:
I would love to know if there was a way to loop through the tabs and add the templates instead of having to hardcode the names in like this. I tried `v-for`s in various places but either got a double up tab bar or a big Vue error. I would imagine at some point we will need to dynamically add tabs based on something.

I have an idea for a PR on kongponents that should be backwards compatible if its not currently possible.

Reply:
This has been amended following a suggestion during review, these are now dynamically generated with a couple of loops 🎉 

KTabs could still do with a tiny bit of massaging to make it work with hashes or with just normal path segments/anchors, but for the moment I got far enough so that I'm happy enough with whilst still continuing to use current KTabs which is what I wanted to do.

Reply:
(Just FYI, this was the PR that was meant to provide better support in KTabs for links: https://github.com/Kong/kongponents/pull/1251)